README.md

```diff
@@ -417,7 +417,7 @@ Users can add a name and optionally a public key. If `public_key` is not provide
 
 #### vsi variable
 
-Note - You can't make changes to the VSI image with this module. That restriction is in place so that you don't inadvertently create an outage or lose data.
+Note - You can't change the VSI image with this module. That restriction is in place so that you don't inadvertently create an outage or lose data.
```

Comment:
I think this should read as "once the infrastructure has been created, any change to this variable won't apply". The current way it reads imply that it is not possible to change the VSI image at all.

Reply:
@vburckhardt Made a change for your review

<img width="711" alt="image" src="https://user-images.githubusercontent.com/8194208/236569889-35282667-b938-4a92-a64c-4b4800e88cac.png">